cdap-stream-clients/java/README.rst

```diff
@@ -60,7 +65,7 @@ Example::
 
 Creating a Stream
 -----------------
-Create a new Stream with the *stream id* "streamName"::
+Create a new Stream with the *stream-id* "streamName"::
```

Comment:
it's probably backward. Names are in italics. Parameters are in between angle brackets, and in literal. Let me fix these...